src/test/java/seedu/address/logic/commands/NoteCommandTest.java

```diff
@@ -36,6 +63,65 @@ public void execute_validNote_addSuccess() {
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model.addPerson(georgiaStaff);
+
+        Staff toAddNotePerson = georgiaStaff;
+        Staff expectedPerson = new Staff(toAddNotePerson.getName(), toAddNotePerson.getPhone(),
+                toAddNotePerson.getEmail(), toAddNotePerson.getAddress(),
+                toAddNotePerson.getTags(), toAddNotePerson.getSalary(), toAddNotePerson.getEmployment());
+        expectedPerson.setNoteContent(validNote1.toString());
+        expectedModel.addPerson(expectedPerson);
+
+
+        NoteCommand noteCommand = new NoteCommand(toAddNotePerson.getName(), validNote1);
+        String expectedMessage = String.format(NoteMessages.MESSAGE_ADD_NOTE_SUCCESS, expectedPerson);
+
+        assertCommandSuccess(noteCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_validNoteSupplier_addSuccess() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model.addPerson(georgiaSupplier);
+
+        Supplier toAddNotePerson = georgiaSupplier;
+        Supplier expectedPerson = new Supplier(toAddNotePerson.getName(), toAddNotePerson.getPhone(),
+                toAddNotePerson.getEmail(), toAddNotePerson.getAddress(),
+                toAddNotePerson.getTags(), toAddNotePerson.getProduct(), toAddNotePerson.getPrice());
+        expectedPerson.setNoteContent(validNote1.toString());
+        expectedModel.addPerson(expectedPerson);
+
+
+        NoteCommand noteCommand = new NoteCommand(toAddNotePerson.getName(), validNote1);
+        String expectedMessage = String.format(NoteMessages.MESSAGE_ADD_NOTE_SUCCESS, expectedPerson);
+
+        assertCommandSuccess(noteCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_validNoteMaintainer_addSuccess() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        model.addPerson(georgiaMaintainer);
+
+        Maintainer toAddNotePerson = georgiaMaintainer;
+        Maintainer expectedPerson = new Maintainer(toAddNotePerson.getName(), toAddNotePerson.getPhone(),
+                toAddNotePerson.getEmail(), toAddNotePerson.getAddress(),
+                toAddNotePerson.getTags(), toAddNotePerson.getSkill(), toAddNotePerson.getCommission());
+        expectedPerson.setNoteContent(validNote1.toString());
+        expectedModel.addPerson(expectedPerson);
+
+
+        NoteCommand noteCommand = new NoteCommand(toAddNotePerson.getName(), validNote1);
+        String expectedMessage = String.format(NoteMessages.MESSAGE_ADD_NOTE_SUCCESS, expectedPerson);
+
+        assertCommandSuccess(noteCommand, model, expectedMessage, expectedModel);
+    }
```

Comment:
I think you can test for execution failures, you can see the edit tests for how I tested for failures.